--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,8 +2,15 @@
 
 project(AppImageLauncher)
 
+`
 # versioning
-include(cmake/versioning.cmake)
+#include(cmake/versioning.cmake)
+set(V_MAJOR 2)
+set(V_MINOR 1)
+set(V_PATCH 0)
+set(V_SUFFIX "")
+
+set(APPIMAGELAUNCHER_VERSION ${V_MAJOR}.${V_MINOR}.${V_PATCH}${V_SUFFIX})
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/cmake/modules)
 
